Index: contrib/minizip/iowin32.c
===================================================================
--- contrib/minizip/iowin32.c	(revision 569)
+++ contrib/minizip/iowin32.c	(working copy)
@@ -27,7 +27,7 @@
 
 
 #if defined(WINAPI_FAMILY_PARTITION) && (!(defined(IOWIN32_USING_WINRT_API)))
-#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+#if WINAPI_FAMILY_ONE_PARTITION(WINAPI_FAMILY_DESKTOP_APP, WINAPI_PARTITION_APP)
 #define IOWIN32_USING_WINRT_API 1
 #endif
 #endif
